serverless/knative_eventing/serverless-kafka.adoc

```diff
@@ -7,6 +7,9 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 You can use the `KafkaChannel` channel type and `KafkaSource` event source with {ServerlessProductName}.
+
+Using Kafka components in your event-driven architecture provides _at least once_ guarantees for event delivery. This means that operations are retried until a return code value is received. However, while this makes your application more resilient to lost events, it may result in duplicate events being sent.
```

Comment:
I'm not a English expert but.. what about ... 

```suggestion
Using Kafka components in your event-driven architecture provides _at least once_ guarantees for event delivery. It means that operations are retried until a return code value is received. However, while this makes your application more resilient to lost events, it may result in duplicate events being sent.
```

Also I was just thinking if this sentence would fit better in as a **Note** section (maybe not...)

Reply:
I think it's OK as is 🙂